src/containers/Alerts/index.js

```diff
@@ -41,28 +43,32 @@ function AlertsComponent(props) {
 
 	return (
 		<div className="alerts" >
-			{props.alerts.map((alert) => {
+			{alerts.map((alert) => {
 				return (
 					<Alert
 						key={alert.key}
 						color={alert.level}
 						className="shadow alerts-style"
 						fade={true}
 						isOpen={!alert.acked}
-						toggle={() => store.dispatch({ type: ACK_ALERT, key: alert.key })}
+						toggle={!sessionExpired ? () => store.dispatch({ type: ACK_ALERT, key: alert.key }) : null} // remove the close button for alert with expiration
 					>
 						{alert.shouldBeTranslated ? t(alert.message) : alert.message}
+						{" "}
+						{sessionExpired &&
+							<a
+								className="alert-link"
+								onClick={() => window.location.reload(false)} // refreshes the page
```

Comment:
@Aringoaway You dont need to add `false` prop to `window.location.reload()` - it has no use